strictdoc/commands/manage_autouid_command.py

```diff
@@ -91,3 +109,69 @@ def execute(
                 document_meta.input_doc_full_path, "w", encoding="utf8"
             ) as output_file:
                 output_file.write(document_content)
+
+        for (
+            trace_info_
+        ) in traceability_index.get_file_traceability_index().trace_infos:
+            assert trace_info_.source_file is not None
+            # FIXME: These conditions for skipping the writes may be insufficient.
+            if (
```

Comment:
From here on, the code IIUC is dedicated to rewrite a single source code file. Maybe refactor it into a method `ManageAutoUIDCommand.rewrite_source_file()`?